### PR TITLE
Switch to reusable workflows for codeception, php-style, poedit…

### PR DESCRIPTION
**Pull request by robertSt7**

### CODE REVIEW

File: .github/workflows/php-cs-fixer.yaml [hunk 1]
```diff
@@ -17,19 +17,9 @@ permissions:
...
-                  repository: ${{ github.event.pull_request.head.repo.full_name }}
-
-            - name: PHP-CS-Fixer
-              uses: docker://oskarstark/php-cs-fixer-ga:latest
-
-            - uses: stefanzweifel/git-auto-commit-action@v5
-              with:
-                  commit_message: Apply php-cs-fixer changes
+    php-style:
+        uses: pimcore/workflows-collection-public/.github/workflows/reusable-php-style.yaml@main
```
Comment (blankse) [inline .github/workflows/php-cs-fixer.yaml]: @robertSt7 Do this workflow also support pull requests from forks? See https://github.com/pimcore/admin-ui-classic-bundle/pull/517

Comment (herbertroth) [reply .github/workflows/php-cs-fixer.yaml]: @blankse It supports pull request from forks(now).

**Review by herbertroth — APPROVED** [review]

File: .github/workflows/php-cs-fixer.yaml [hunk 2]
```diff
@@ -17,19 +17,10 @@ permissions:
     contents: read
```
Comment (blankse) [inline .github/workflows/php-cs-fixer.yaml]: Must it be `write`?
```suggestion
    contents: write
```

File: .github/workflows/php-cs-fixer.yaml [hunk 3]
```diff
@@ -17,19 +17,10 @@ permissions:
...
-            - uses: stefanzweifel/git-auto-commit-action@v5
-              with:
-                  commit_message: Apply php-cs-fixer changes
+    php-style:
+        uses: pimcore/workflows-collection-public/.github/workflows/reusable-php-style.yaml@main
+        secrets:
+            PHP_CS_FIXER_GITHUB_TOKEN: ${{ secrets.PHP_CS_FIXER_GITHUB_TOKEN }}
+        with:
+            head_ref: ${{ github.ref }}
+            config_file: .php-cs-fixer.dist.php
```
Comment (blankse) [inline .github/workflows/php-cs-fixer.yaml]: Is here `repository` missing?
```suggestion
            head_ref: ${{ github.event.pull_request.head.ref }}
            repository: ${{ github.event.pull_request.head.repo.full_name }}
            config_file: .php-cs-fixer.dist.php
```

**Comment by blankse** [comment]
@robertSt7 @herbertroth My suggestions are from the file in `pimcore/pimcore`
https://github.com/pimcore/pimcore/blob/11.x/.github/workflows/php-cs-fixer.yaml

When I update my fork of `pimcore/pimcore` the php-cs-fixer action in the fork are failing:
https://github.com/blankse/pimcore/actions/runs/9748030425/job/26902090652

Is it possible to avoid this?

**Pull request merged**